rustyms/src/shared/glycan.rs

```diff
@@ -568,10 +625,24 @@ pub enum HeptoseIsomer {
     Sedoheptulose,
 }
 
+/// Any 9 carbon glycan, these isomers are modification specific (need the correct substituents applied to be meaningful)
+#[allow(dead_code)]
+#[derive(Clone, Eq, PartialEq, Ord, PartialOrd, Hash, Debug, Serialize, Deserialize)]
+pub enum NonoseIsomer {
```

Comment:
This kind of classification is imprecise. What if you have a Kdn with an additional modification or substitution. Does it cease to be `Kdn` or is it `Kdn` + something? Admittedly, these are special cases, but they are only "special" in so far as IUPAC trivial names are concerned.

Reply:
Similarly, `NeuAc` is actually `2-keto-3-deoxy-Nonp5N` with an acidic modification at the first position, plus an `acetyl` added to the amine group at the fifth backbone position. Additionally, `Neu5Ac8Ac` only adds another `acetyl`, not another acetylosamine. 

I haven't used the formal names in ~5 years so the precise spelling could be off.